src/client.c

```diff
@@ -31,23 +31,30 @@ urlinfo_t *parse_url(char *url)
   char *hostname = strdup(url);
   char *port;
   char *path;
+  char *backlash, *colon;
 
   urlinfo_t *urlinfo = malloc(sizeof(urlinfo_t));
-
-  /*
-    We can parse the input URL by doing the following:
-
-    1. Use strchr to find the first backslash in the URL (this is assuming there is no http:// or https:// in the URL).
-    2. Set the path pointer to 1 character after the spot returned by strchr.
-    3. Overwrite the backslash with a '\0' so that we are no longer considering anything after the backslash.
-    4. Use strchr to find the first colon in the URL.
-    5. Set the port pointer to 1 character after the spot returned by strchr.
-    6. Overwrite the colon with a '\0' so that we are just left with the hostname.
-  */
-
-  ///////////////////
-  // IMPLEMENT ME! //
-  ///////////////////
+  
+  // We can parse the input URL by doing the following:
+  
+  // 1. Use strchr to find the first backslash in the URL (this is assuming there is no http:// or https:// in the URL).
+  backlash = strchr(hostname, '/');
+  // 2. Set the path pointer to 1 character after the spot returned by strchr.
+  path = backlash + 1;
```

Comment:
You would then add three to the value returned from the call to strstr() above if you did it my way. If you choose to do it this way though, I think it can still work, but you want to do path = backlash + 2 because the forward slashes will occur together and you need "jump over" both of them.